app/config/Seeds/ArticlesSeed.php

```diff
@@ -0,0 +1,61 @@
+<?php
+declare(strict_types=1);
+
+use Cake\Log\Log;
+use Phinx\Seed\AbstractSeed;
+
+/**
+ * Articles seed.
+ */
+class ArticlesSeed extends AbstractSeed
+{ 
+    public function getDependencies(): array
+    {
+        return [
+            'UsersSeed',
+        ];
+    }
+
+    /**
+     * Run Method.
+     *
+     * Write your database seeder using this method.
+     *
+     * More information on writing seeds is available here:
+     * https://book.cakephp.org/phinx/0/en/seeding.html
+     *
+     * @return void
+     */
+    public function run(): void
+    {
+        try {
```

Comment:
From comment https://github.com/ArielGalvez/tutorial-cakephp/pull/1#discussion_r1386859350, the try & catch is no longer necessary once we check the data existence.

Usually, we use try & catch when we expect to handle/deal the exception (like your previous approach to catch the exception thrown by PDO Exception). We don't want the program terminated abruptly due to improper handling.

Anyway, the current try & catch approach is acceptable. In the future, please think first whether it's necessary to add try & catch.